static/src/javascripts/projects/common/modules/experiments/ab-test-clash.js

```diff
@@ -13,10 +13,10 @@ define([
     function _testABClash(f) {
 
         var contributionsEpic = {name: 'ContributionsEpic20160916', variants: ['control', 'give', 'today', 'make']};
-
-        var contributionsEpicButtons = {name: 'ContributionsEpicButtons20160907', variants: ['control', 'buttons']};
-
-        var clashingTests = [contributionsEpic, contributionsEpicButtons];
+        
```

Comment:
I just meant there was random white space that was probably added by your text editor and that you might want to remove it.